config/features/flags.go

```diff
@@ -41,9 +41,9 @@ var (
 		Name:  "disable-grpc-connection-logging",
 		Usage: "Disables displaying logs for newly connected grpc clients",
 	}
-	enableReorgLateBlocks = &cli.BoolFlag{
-		Name:  "enable-reorg-late-blocks",
-		Usage: "Enables reorgs of late blocks",
+	disableReorgLateBlocks = &cli.BoolFlag{
+		Name:  "disable-reorg-late-blocks",
+		Usage: "Disables reorgs of late blocks",
```

Comment:
Let's remember to update this for our testnet nodes

Reply:
nice catch